api/models/User.js

```diff
@@ -59,7 +59,7 @@ const UserSchema = new Schema(
     // 2: Admin
     accessLevel: {
       type: Number,
-      default: 2
+      default: -1
```

Comment:
no more hardcoded numbers. [we have enums for this](https://github.com/SCE-Development/Core-v4/blob/3c4ba959ba6a462c28bae5841594edf1ed625033/src/Enums.js#L28) so import them